templates/patterns/list_item/list_item.ui_patterns.yml

```diff
@@ -76,3 +76,12 @@ list_item:
       preview:
         src: "https://placeimg.com/1200/900/arch"
         alt: "Alternative text for image"
+    field_list:
+        type: "array"
+        label: "Items"
+        description: "Label of the field"
```

Comment:
Label and description are not correct, let's use:
```
        label: "List of fields"
        description: "List of fields to be displayed under the item description. Only used on XXX"
```
The use the variants where this is used instead of `XXX`

Reply:
Updated